ListLinked/LinkedList.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace LinkedList
+{
+    
+    public  class LinkedList<T> : IEnumerable<T>
+    {
+        private Node<T> _head;
+        public int Length { get; private set; }
+
+        private Node<T> LastAdd { get; set; }
```

Comment:
Should be private with a right name. How does _tail look for you?

Reply:
Add _